financial-connections/src/main/java/com/stripe/android/financialconnections/features/common/LoadingContent.kt

```diff
@@ -137,7 +160,84 @@ internal fun LoadingSpinner(
     }
 }
 
-private const val LOADING_SPINNER_ROTATION_MS = 1000
+@Composable
+internal fun LoadingPill(
+    canShowPill: Boolean,
+    modifier: Modifier = Modifier,
+    showPillImmediately: Boolean = false,
```

Comment:
In which situations do we want to show it immediately (besides in previews)? If there's none we can just rely on `inspectionMode`.

Reply:
Yep, I’m removing `showPillImmediately`.